core/modules/flowgen.cc

```diff
@@ -206,6 +209,18 @@ pb_error_t FlowGen::ProcessArguments(const bess::pb::FlowGenArg &arg) {
     quick_rampup_ = 1;
   }
 
+  ip_src_range_ = arg.ip_src_range();
+  ip_dst_range_ = arg.ip_dst_range();
+  port_src_range_ = (uint16_t)arg.port_src_range();
```

Comment:
Port types should be uint16 in .proto file in the first place, rather than doing type casting here.